libtransmission/variant.cc

```diff
@@ -55,7 +55,7 @@ static void tr_variant_string_clear(struct tr_variant_string* str)
 {
     if (str->type == TR_STRING_TYPE_HEAP)
     {
-        delete[]((char*)(str->str.str));
+        delete[] ((char*)(str->str.str));
```

Comment:
This is a sign that you've been skipping the so-called "pre-commit" hook.
You need to use clang-format to validate your code formatting. Or, in short, you can't add a space here.

Reply:
Actually, that's how clang-format v15 lays it out.

Reply:
Ah, mea culpa. I'm on clang-format@12 for some reason.

Reply:
Yes, I upgraded to clang-format-15. Because my last version was clang-format-10, which lacks some features.
Can I use it?